src/content/docs/03-var/02_path-sep.md

```diff
@@ -9,7 +9,7 @@ sidebar:
 The `path.sep` property is a platform-specific path segment separator used in file system paths. It provides a string representing the character used to separate directories or folders in a file path. The value of `path.sep` varies depending on the operating system.
 
 :::note
-On *Unix-based* systems (Linux, macOS), the path separator is `/`.
+On *Unix-based* systems (Linux, macOS), the path separator is `/`.<br>
```

Comment:
Please try to avoid changing files that are not in the scope of the PR. If you think that you want to change this file, it is fine, and i say thank you, but please open a new specific PR.

Reply:
Sure Roberto. Let me revert the commit and make necessary changes and create a new PR.